stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -75,20 +75,25 @@ class Stripe internal constructor(
      * @param enableLogging enable logging in the Stripe and Stripe 3DS2 SDKs; disabled by default.
      * It is recommended to disable logging in production. Logs can be accessed from the command line using
      * `adb logcat -s StripeSdk`
+     * @param betas optional, set of beta flags to pass to the Stripe API. Setting this property is
+     * not sufficient to participate in a beta, and passing a beta you are not registered
+     * in will result in API errors.
```

Comment:
Out of curiosity, is this a string we use for other clients?

Reply:
at least also for iOS client